core/src/main/java/io/javaoperatorsdk/webhook/admission/mutation/AsyncDefaultAdmissionRequestMutator.java

```diff
@@ -15,37 +15,48 @@
 
 import static io.javaoperatorsdk.webhook.admission.AdmissionUtils.admissionResponseFromMutation;
 import static io.javaoperatorsdk.webhook.admission.AdmissionUtils.getTargetResource;
+import static io.javaoperatorsdk.webhook.admission.AdmissionUtils.notAllowedExceptionToAdmissionResponse;
 
 public class AsyncDefaultAdmissionRequestMutator<T extends KubernetesResource>
     implements AsyncAdmissionRequestHandler {
 
-  private final AsyncMutator<T> mutator;
+  private final AsyncMutator<T> asyncMutator;
   private final Cloner<T> cloner;
 
-  public AsyncDefaultAdmissionRequestMutator(AsyncMutator<T> mutator) {
+  public AsyncDefaultAdmissionRequestMutator(Mutator<T> mutator) {
     this(mutator, new ObjectMapperCloner<>());
   }
 
-  public AsyncDefaultAdmissionRequestMutator(AsyncMutator<T> mutator, Cloner<T> cloner) {
-    this.mutator = mutator;
+  public AsyncDefaultAdmissionRequestMutator(Mutator<T> mutator, Cloner<T> cloner) {
```

Comment:
This is the internal convenience constructor to enable the use of a `Mutator` for an async endpoint.